code/datums/supply_packs/operations.dm

```diff
@@ -1,117 +1,151 @@
-//the idea is to put all the bulk items scanner secure crate with lot's of flares MRE in it and at the end OB and non buyable.
+//large item crates
 
-//non buyable
+/datum/supply_packs/floodlight
+	name = "floodlight crate (x4)"
+	contains = list(
+					/obj/structure/machinery/floodlight,
+					/obj/structure/machinery/floodlight,
+					/obj/structure/machinery/floodlight,
+					/obj/structure/machinery/floodlight,
+					)
+	cost = 20
+	containertype = /obj/structure/largecrate
+	containername = "\improper Floodlights Crate"
+	group = "Operations"
+
+/datum/supply_packs/powerloader
+	name = "powerloader crate"
+	contains = list(
+					/obj/vehicle/powerloader,
+					/obj/item/pamphlet/skill/powerloader,
+					)
+	cost = 20
+	containertype = /obj/structure/largecrate
+	containername = "\improper Powerloader Crate"
+	group = "Operations"
 
-/datum/supply_packs/ob_incendiary
+/datum/supply_packs/bulk_mre
+	name = "USCM MRE crate (x60)"
 	contains = list(
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/warhead/incendiary,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/ob_fuel,
-		/obj/structure/ob_ammo/warhead/incendiary,
-	)
+					/obj/item/ammo_box/magazine/misc/mre,
+					/obj/item/ammo_box/magazine/misc/mre,
+					/obj/item/ammo_box/magazine/misc/mre,
+					/obj/item/ammo_box/magazine/misc/mre,
+					/obj/item/ammo_box/magazine/misc/mre,
+					)
+	cost = 20
+	containertype = /obj/structure/largecrate
+	containername = "\improper USCM MRE Crate (x60)"
+	group = "Operations"
 
-	name = "OB Incendiary Crate"
+/datum/supply_packs/bulk_flares
+	name = "Flare supply crate (x200)"
+	contains = list(
+					/obj/item/ammo_box/magazine/misc/flares,
+					/obj/item/ammo_box/magazine/misc/flares,
+					)
+	cost = 20
+	containertype = /obj/structure/largecrate
+	containername = "\improper Flare Supply Crate (x200)"
+	group = "Operations"
+
+//can't buy them crate for staff to spawn in events.
+
+/datum/supply_packs/staff
```

Comment:
Made a parent object for non spawn able object  but was unable to test it asked on discord no answer on how to admin spawn in ASRS list...